doc-content/enterprise-only/bpmn/bpmn-con.adoc

```diff
@@ -1,8 +1,63 @@
 [id='bpmn-con_{context}']
 = Business Process Modeling and Notation Version 2.0
-The Business Process Modeling and Notation Version 2.0 (BPMN2) specification is an Object Management Group (OMG) specification that defines standards for graphically representing a business process, defines execution semantics for the elements, and provides process definitions in XML format. 
+The Business Process Modeling and Notation Version 2.0 (BPMN2) specification is an Object Management Group (OMG) specification that defines standards for graphically representing a business process, defines execution semantics for the elements, and provides process definitions in XML format.
 
-A process is defined or determined by its process definition. It exists in a knowledge base and is identified by its ID. A  process is a container for a set of modeling elements. It contains elements that specify the execution workflow of a business process or its parts using flow objects and flows. Each process has its own BPMN2 diagram. {PRODUCT} contains the legacy process designer and the new process designer for creating BPMN2 diagrams. The new process designer has an improved layout and feature set and continues to be developed. Until all features of the legacy process designer are completely implemented in the new process designer, both designers are available in {CENTRAL} for you to use.
+A process is defined or determined by its process definition. It exists in a knowledge base and is identified by its ID or other properties.
 
+--
+.General process properties
+[cols="30%,70%", options="header"]
+|===
+|Label
+|Description
 
+| *Name*
+| Enter the name of the process.
 
+| *Documentation*
+| Describes the process. The text in this field is included in the process documentation, if applicable.
+
+| *ID*
+| Enter an identifier for this process, such as `orderItems`.
+
+| *Package*
+| Enter the package location for this process in your {PRODUCT} project, such as `org.acme`.
+
+| *ProcessType*
+| Specify whether the process is public or private (or null, if not applicable).
+
+| *Version*
+| Enter the artifact version for the process.
+
+| *Ad hoc*
+| Select this option if this process is an ad hoc subprocess. (Currently not supported.)
```

Comment:
@lazarotti, @romartin, can you please provide your suggestion?

Reply:
well it's true that we're mostly considering AdHoc subprocesses as for our case maangement implementation. But anyway it does not means that just setting this flag we're talking about CM in BC, because also the process/case needs to be in a case project, otherwise it's just a "normal" process.